dss/storage/checkout/cache_flow.py

```diff
@@ -19,8 +19,8 @@
 
 
 def is_dss_bucket(dst_bucket: str):
-    return dst_bucket in (os.environ['DSS_GS_CHECKOUT_BUCKET_TEST'], os.environ['DSS_S3_CHECKOUT_BUCKET_TEST'],
-                          os.environ['DSS_S3_CHECKOUT_BUCKET'], os.environ['DSS_GS_CHECKOUT_BUCKET'])
+    """Lambdas only have access to the standard checkout buckets, should not modify non-DSS buckets"""
```

Comment:
This docstring seems out of context (what the function does and why), is this function necessarily directly associated with the lambdas?